packages/gatsby-plugin-netlify/src/build-headers-program.js

```diff
@@ -28,7 +29,7 @@ function linkTemplate(assetPath, type = `script`) {
 }
 
 function pathChunkName(path) {
-  const name = path === `/` ? `index` : _.kebabCase(path)
+  const name = path === `/` ? `index` : kebabHash(path)
```

Comment:
This actually made me realize, that netlify http/2 push most defenitely doesn't work for json data with gatsby v2, as we don't use webpack for that anymore.

Reply:
Re my previous comment - this isn't working correctly currently in v2, so let's not block this PR with this